app/views/posts/show.html.erb

```diff
@@ -1 +1,13 @@
-<h2>This is the show page for posts</h2>
+<h1> show post for a given user</h1>
+<div>
+  <div>
+    <h3> <%=@post.title%> by <%= @user.name%> </h3>
+    <p> <%= @post.text%> </p>
+    <p> Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %> </p>
+  </div>
+  <div>
+    <% @post.recent_five_comments.each do |comment| %>
```

Comment:
- [ ] I get this error when I visit the post show page. Kindly fix that. What you have is `last_five_comments`, not `recent_five_comments`.

  ![didow](https://user-images.githubusercontent.com/54848363/205268889-baac972f-acb9-457e-a150-68e2a2347ee6.jpg)